### PR TITLE
Travis CI config

### PR DESCRIPTION
**Pull request by steve-taylor**
This PR adds Travis CI config for automated deployment to npmjs.

Before merging this PR, replace my `email` and `api_key` with [yours](https://docs.travis-ci.com/user/deployment/npm/#npm-auth-token).

Before tagging for the first time, sign in to [Travis CI](https://travis-ci.org/) and add this repo.

Bump the package version before tagging, because version 0.0.1 is already [published](https://www.npmjs.com/package/vizard).

### CODE REVIEW

**Comment by steve-taylor** [comment]
Bump the version before tagging, because I took the liberty of [publishing](https://www.npmjs.com/package/vizard) version 0.0.1.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+cache:
+  directories:
+  - node_modules
+script: 'true'
+deploy:
+  skip_cleanup: true
+  provider: npm
+  email: mail@stevetaylor.io
+  api_key:
+    secure: YcmT6C5vaXZhRmJglcDEBrme6ew+q65jA0ZFZi9Zd//cLCYK5Aq0jZXy38f463iDqloHjZ36U94wFfRWP4rWUHGZM8fswcTNGBjvTNgOT53mWTl46gg+0lcYV7/g1csO3ZKsy3nG1VkJSF1COf96YImQqgkSx5zWEeAnNrV68edAMLpbu2EVqNihYLO9sfpcEmq4yj9mwY0Muwrim0Xo+0dop0EKLRzcPITplNkkF3l4wexb60M1fhP7I3ZRZjCVcaeBj3W/OfJ/SjVI/D30P4Do43KVCK7okHUK4R72vi8ZWTOmZnVXiqQ1MAPLry0jGs8k77++AKhzn9g7Y5OeB4YmTlz7toqLuLFhXxuZBiAHyCyQpv4xZSTWaZNWh/KeFJkRe/bXSQJdG885+mrWouG3I8UQzHWhIs3cGEF/8N3IYEZC4swUw9PQUSwJFcxe0+BSgF1ZpuvsbzFcedAgxorC6kLMTKUdEsCcLTf8I9MTBBn19QYBBAmuSaKJ6A9XeybV6k40kz6wyM8e+j9mMluYdLM0L5xtalr8EQk5hZSFzCdBshNDDMXngUeIRhMOyOXbUly/Tt2QfVmE0RVeaeuC5GljCMihrOB1P4RwJ1plgsVOPc0sJkkzFwTEWNLHxnhN4JNy9kBbgJu+/SIC3ap+4XVL1+amDiSIxqMx9oY=
```
Comment (archinal) [inline .travis.yml]: We're not quite ready to start building this on travis at the moment, so can we merge this PR without the `.travis.yml`?

The rest all looks good

Comment (steve-taylor) [reply .travis.yml]: Hi @archinal. You might as well close this PR because it's all about Travis. I can remove Travis from the other PR, though.

Comment (archinal) [reply .travis.yml]: The change of `serve` provider is worth merging in - github has complained about the security vulnerabilities too

Comment (steve-taylor) [reply .travis.yml]: The change from old `serve` to new `serve-handler` is in #3.

**Pull request closed without merging**